content/en/blog/falco-okta-identity/index.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Using Falco to Create Custom Identity Detections
+linktitle: Understanding Okta plugin for Falco
+description: Learn how Falco plugin is used for Identity Threat Detection
+date: 2023-11-28
+author: Nigel Douglas
+slug: falco-okta-identity
+images:
+  - /blog/falco-okta-identity/images/falco-okta-featured.png
+tags: ["Falco", "Okta", "Plugin", "Identity"]
```

Comment:
here's the list of tags you can use https://github.com/falcosecurity/falco-website/tree/master/data/canonical-tags